### PR TITLE
Added router

### LINKED ISSUE #13: Setup react router for the website
This is not too difficult, but it may require some understanding of `react-router`. [This](https://github.com/salahusama/WorkflowStreamer-UI/blob/master/workflowstreamer-ui/src/Router.js) is one I had setup on another project.

### PR DESCRIPTION
**Pull request by salahusama**
Closes #13 
* Added react router
  * This allows us to go to a component based on the url
* Added a placeholder admin page as an example to show routing

### Usage
* `/` will automatically redirect to `/app`
* `/app` will show the `App` component
* `/admin` will show the admin page

### CODE REVIEW

File: src/Router.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import App from './components/App';
+import AdminPage from './components/AdminPage';
+
+export default function Router() {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Redirect exact from="/" to="/app" />
```
Comment (hashdoge) [inline src/Router.js]: for clarification, this would work like "hostname:port/" => "hostname:port/app" ?

Comment (salahusama) [reply src/Router.js]: Yeah. We can change paths later. We can just have the `/` go to app and not redirect.

**Comment by salahusama** [comment]
@mahmoudal99, any chance you can look at this?

**Comment by mahmoudal99** [comment]
> @mahmoudal99, any chance you can look at this?

I will look into it now and get back to you asap

**Pull request merged**